CODE_OF_CONDUCT.md

```diff
@@ -1,13 +1,3 @@
-# Community Code of Conduct
+# Overview
 
-## Overview
-
-This document makes a reference to the rules of behavior for all contributors and maintainers in the community. All members agree to follow them to foster the growth of the community of tolerance, respect, and mutual understanding.
-
-## Code of conduct
-
-In the attempt to adopt the best practices from the most renowned open-source and cloud-native projects, and work closely with the Cloud Native Computing Foundation (CNCF), our community members undertake to follow the rules outlined in the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Read it to learn about the harassment-free environment, examples of unacceptable behavior, and the responsibilities of the project maintainers.
-
-## Violations
-
-If you observe any instance of abusive, harassing, or otherwise unacceptable behavior, report it as an issue to the project maintainers specified in the `CODEOWNERS` document.
+To to read about the Code of Conduct of this project, go to the [`CODE_OF_CONDUCT.md`](/contributing/CODE_OF_CONDUCT.md) document.
```

Comment:
```suggestion
To read about the Code of Conduct of this project, go to the [`this`](/contributing/01-code-of-conduct.md) document.
```

Reply:
Done, I just wouldn't put "this" in code.

Reply:
Yup, my bad.